packages/@okta/vuepress-site/docs/reference/api/authn/index.md

```diff
@@ -871,19 +871,24 @@ Content-Type: application/json
 
 Include the `X-Device-Fingerprint` header to supply a device fingerprint. The device fingerprint is used in the following ways:
 
-* If the new or unknown-device email notification is enabled, an email is sent to the user if the device fingerprint sent in the header is not associated with a previously successful user sign in. For more information about this feature, see the [General Security documentation](https://help.okta.com/en/prod/Content/Topics/Security/Security_General.htm?).
-* If you have the security behavior detection feature enabled and you have a new device behavior configured in a policy rule, a new device is detected if the device fingerprint sent in the header is not associated with previous successful logins. For more information about this feature, see [EA documentation](https://help.okta.com/en/prod/Content/Topics/Security/proc-security-behavior-detection.htm?).
+* If the new or unknown device email notification is enabled, an email is sent to the user if the device fingerprint sent in the `X-Device-Fingerprint` header isn't associated with a previously successful user sign in. For more information about this feature, see the [General Security documentation](https://help.okta.com/en/prod/Content/Topics/Security/Security_General.htm?).
+* If you have the security behavior detection feature enabled and you have a new device behavior configured in a policy rule, a new device is detected if the device fingerprint sent in the `X-Device-Fingerprint` header isn't associated with a previously successful user sign in. For more information about this feature, see [EA documentation](https://help.okta.com/en/prod/Content/Topics/Security/proc-security-behavior-detection.htm?).
```

Comment:
Should we remove this bullet? This is the behavior we agreed is ultimately wrong and we are going to change, per the rollout plan here: https://oktawiki.atlassian.net/wiki/spaces/PM/pages/734406988/Fixing+Known+Device+behavior+for+V1+IDX#Fixing%22KnownDevice%22behaviorforV1&IDX-Rolloutplan

> Change doc to describe correct use of X-Device-Token (remove X-Device-Fingerprint from docs)

Although we haven't fixed the product behavior yet, I think we should either remove this or mark it with the `Deprecated` tag and explain that this use case is going to be superseded by passing Device Token instead (and link to the later section about Device Token).

Reply:
If a customer enabled New device behavior detection though they would need to pass this header in. So I think we may need to wait until we have the new solution before we can Deprecate this.